.github/workflows/pypipublish_linux.yml

```diff
@@ -0,0 +1,67 @@
+name: Linux Package PyPI deploy
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    strategy:
+      matrix:
+        os: 
+          - ubuntu-latest
+        python-version: [3.5, 3.6, 3.7,3.8]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up submodules
+        run: |
+          bash ./ext_source_setup.sh
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install pre-requisites
+        run: |
+          sudo apt-get update
+          # checking for g++
+          dpkg -s g++ &> /dev/null
+          if [ $? -eq 0 ]; then
+              echo "g++ is installed, skipping..."
+          else
+              echo "Installing g++"
+              sudo apt-get install g++
+          fi
+          echo "Installing Bazel dependencies"
+          sudo apt-get install pkg-config zip zlib1g-dev unzip
+          echo "Donwloading Bazel 2.1.0"
+          wget https://github.com/bazelbuild/bazel/releases/download/2.1.0/bazel-2.1.0-installer-linux-x86_64.sh
+
+          chmod +x bazel-2.1.0-installer-linux-x86_64.sh
+          ./bazel-2.1.0-installer-linux-x86_64.sh --user
+          export PATH="$PATH:$HOME/bin"
+      - name: Build pydp lib
+        run: |
+          bash ./build_PyDP.sh
+
+      - name: Install python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r ./requirements_dev.txt
+
+      - name: Build wheel
+        run: |
+          python setup.py bdist_wheel
+
+      - name: Renaming wheel
+        run: |
+          find . -name '*linux*.whl' -type f -exec bash -c 'mv "$1" "${1/linux/manylinux1}"' -- {} \;
```

Comment:
I don't agree that renaming is the way to go here, you should be using the auditwheel package for compliance, it might break on some systems.

Reply:
The package is manylinux1 compliant. So adding it here. 
But in the meantime, we are in the process of looking for a permanent solution.

Reply:
Hi there ,

@chinmayshah99 let’s open an issue as it is a good practise to normally use auditwheel to publish on PyPi.

It will enable to keep track or the changes.

Reply:
Yeah, let's do this. Are you creating an issue?